size

```diff
@@ -0,0 +1,3 @@
+size2x=$(xxd -p pinpog | tr -d '\n' | pcregrep -o1 -i '^([0-9a-f]+?)[c]*55aa$' | wc -c)
+echo $(($size2x / 2))
+
```

Comment:
I don't like this approach. It adds additional dev dependencies and requires additional documentation on how to use this script.

Is it possible to just print the size of the resulting executable as part of the compilation log? I mean NASM already knows the value of `$-$$`, right? Why can't it just print it for us?

Reply:
I didn't find any way of doing this in NASM. But I know the way to do it in FASM. I can make a PR which will convert the source code to FASM.

Reply:
> I didn't find any way of doing this in NASM.

Me too. This is why I created #32. :)

> I can make a PR which will convert the source code to FASM.

Sounds like a good content. Let me do it myself on the stream.

P.S. Please don't close the PR until the next stream. We may use it as a plan B.

Reply:
I created a [minimal example](https://gist.github.com/RIscRIpt/a915c6a2f81f862a5323e9d996596196), which you can use to get size before and after compilation.